cpp/src/parquet.cpp

```diff
@@ -129,7 +131,7 @@ class ParquetRead : public Task<ParquetRead, OpCode::ParquetRead> {
       for (const auto& table : tables) {
         table_views.push_back(table->view());
       }
-      tbl_arg.move_into(cudf::concatenate(table_views, ctx.stream(), ctx.mr()));
+      tbl_arg.move_into(cudf::concatenate(table_views, context.get_task_stream()));
```

Comment:
OK, too many to suggest.  All of these can remain unchanged now, since we still have the `ctx.stream()` short-hand and `ctx.mr()` is available again and must still be passed in.